nixos/modules/services/security/howdy/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.howdy;
+  settingsType = pkgs.formats.ini { };
+in
+{
+  options = {
+    services.howdy = {
+      enable = lib.mkEnableOption "" // {
+        description = ''
+          Howdy and PAM module for face recognition. See
+          `services.linux-enable-ir-emitter` for enabling the IR emitter support.
+        '';
```

Comment:
In this case it won't have "Whether to enable" added

Reply:
Fixed.